lib/vf_settokenvalue.cpp

```diff
@@ -486,17 +486,14 @@ namespace ValueFlow
                             continue;
                         result.valueType = Value::ValueType::FLOAT;
                     }
-                    const double floatValue1 = value1.isFloatValue() ? value1.floatValue : static_cast<double>(value1.intvalue);
-                    const double floatValue2 = value2.isFloatValue() ? value2.floatValue : static_cast<double>(value2.intvalue);
-                    const bool isFloat = value1.isFloatValue() || value2.isFloatValue();
```

Comment:
Why not keep `isFloat` since it's used in two places?

Reply:
It should stay because it can be used in more places. Will adjust.